.github/workflows/release.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       env:
         JAR_PATH: ${{ secrets.JAR_PATH }}
         SERVER_PORT: ${{ secrets.SERVER_PORT }}
-      run: docker build . -f Dockerfile --tag ${{ secrets.DOCKER_REPO }}:${{ github.ref_name }} --build-arg JAR_PATH=$JAR_PATH --build-arg SERVER_PORT=$SERVER_PORT
+      run: docker build . -f Dockerfile --tag ${{ secrets.DOCKER_REPO }}:${{ github.ref_name }} --tag ${{ secrets.DOCKER_REPO }}:latest --build-arg JAR_PATH=$JAR_PATH --build-arg SERVER_PORT=$SERVER_PORT
```

Comment:
Test this and update the documentation also specifying that users can also pull using the specific tag version that is same as the Github's latest release tag. https://github.com/Brightscout/mattermost-plugin-exchange-ews-proxy/blob/54310d4c488ca8bb3809e0f28c4eba2fe8e472e1/README.md?plain=1#L55

Reply:
Tested and updated docs